weave/trace/util.py

```diff
@@ -95,3 +95,32 @@ def run(self) -> None:
 # rename for cleaner export
 ThreadPoolExecutor = ContextAwareThreadPoolExecutor
 Thread = ContextAwareThread
+
+
+def sanitize_call_name(call_name: str) -> str:
+    """Sanitizes calls following the standard set in our Langchain integration.
+
+    - Alphanumeric + period, underscore, hyphen, space, backslash.
```

Comment:
The backslash here is escaping the dash which has a special meaning (range, like in `a-z`) inside the regexp. We don't want to allow backslashes in names.